Gemfile

```
@@ -42,7 +42,7 @@ group :development, :test do
   gem 'database_cleaner'
   gem 'factory_bot_rails', '< 5.0'
 
-  gem 'rubocop', require: false
+  gem 'rubocop', '~> 0.76.0', require: false
```

Why did you change this ?

```suggestion
  gem 'rubocop', require: false
```

---

Changed it just to stop bundle install complaint about the rubocop version (already defined in redmine 4.1x Gemfile).
But I agree it's more a dirty hack than a patch.